tests/acceptance/features/bootstrap/Sharing.php

```diff
@@ -1762,14 +1731,33 @@ public function userHasDeletedLastShareUsingTheSharingApi(string $user):void {
 		$this->theHTTPStatusCodeShouldBeBetween(200, 299, $response);
 	}
 
+	/**
+	 * @param string $user
+	 * @param string $shareType    user|group|link
+	 * @param string|null $language
+	 *
+	 * @return ResponseInterface
+	 */
+	public function getLastShareInfo(string $user, string $shareType, ?string $language = null): ResponseInterface {
+		if ($shareType !== "link") {
+			$shareId = $this->isUsingSharingNg() ? $this->shareNgGetLastCreatedUserGroupShareID() : $this->getLastCreatedUserGroupShareId();
+		} else {
+			$shareId = (string) $this->getLastCreatedPublicShare()->id;
```

Comment:
we do have link share stores both for sharingNG and ocs sharing
```suggestion
			$shareId = (string) $this->getLastCreatedPublicShare()->id;
```